README.md

```
@@ -21,65 +20,79 @@ See https://github.com/roadrunner-server/roadrunner - High-performance PHP appli
+- works on Windows (named pipes, Unix-domain sockets via AF_UNIX)
 
 Installation
 ------------
 
-```go
-GO111MODULE=on go get github.com/roadrunner-server/goridge/v4
+```bash
+go get github.com/roadrunner-server/goridge/v4
 ```
 
 ### Sample of usage
```

markdownlint-cli2 (0.22.1)

[warning] Heading style
Expected: setext; Actual: atx

(MD003, heading-style)